mobile/lib/src/domain/sleep_history/night_stats.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:polydodo/src/domain/entity.dart';
+import 'package:flutter/foundation.dart';
+
+class NightStats extends Entity {
+  final DateTime recordingStart;
+  final DateTime recordingEnd;
```

Comment:
Maybe create a structure for this called `Range<T>` which contains `T start` and `T end`?

Reply:
Bless DateTimeRange.

Reply:
Encore mieux